installing/installing_openstack/installing-openstack-user.adoc

```diff
@@ -0,0 +1,81 @@
+[id="installing-openstack-user"]
+= Installing a cluster on OpenStack on your own infrastructure
+include::modules/common-attributes.adoc[]
+:context: installing-openstack-user
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+{rh-openstack-first} that runs on infrastructure that you provide.
+
+Using your own infrastructure allows you to integrate your cluster with existing infrastructure and modifications. The process requires more labor on your part than installation-program-provisioned installations, however, as you must create all {rh-openstack} resources, like Nova servers, Neutron ports, and security groups.
+
+.Prerequisites
+
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
+* Have access to an {rh-openstack} administrator's account
+
+* On the target installation environment, have a single directory in which you can keep the files you create during the installation process
```

Comment:
What you refer here as "target installation environment" is really the machine from which you run the installer. I find the word "target" a bit confusing, because it can really be your home laptop; when I think "target", I rather think of the machines onto which OpenShift will be installed.